how-to/install/vanilla.md

```diff
@@ -0,0 +1,197 @@
+---
+hidden: true
+---
+
+# Minecraft Launcher
+
+<!-- TODO: do NOT change list points for the manual installation. "Why do I need to copy everything over?" relies on them -->
+
+{% tabs %}
+{% tab title="Installer (recommended)" %}
+{% tabs %}
+{% tab title="Windows" %}
+1. Download the [FO Installer](https://download.fo/vanilla)
+2. Open the `exe` installer. If you get a security prompt from Windows, allow the installer to run:
+   * [Why did I get this prompt?](#why-does-windows-show-the-security-prompt)
```

Comment:
could just have an inline "(why?)" or at least have it as the last bullet point; as first it is confusing